docs/strucpp-migration/01-strucpp-compiler-integration.md

```
@@ -0,0 +1,192 @@
+
+**File**: `.gitignore`
+
+```gitignore
+resources/strucpp/
+```
+
+The `node_modules/strucpp` entry is already covered by the blanket `node_modules/` ignore.
+
+### Directory Layout After Setup
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)